.github/workflows/lint-test.yaml

```diff
@@ -44,4 +44,4 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Run chart-testing (install)
-        run: ct install --config ct.yaml
+        run: ct install --config ct.yaml --excluded-charts neon-storage-controller
```

Comment:
It is common (and I would say necessary) practice to run a helm chart with a default values file. What do you think about adding a dependent postgresql chart with the necessary changes in the values file?

Reply:
We can do that once the service has a way to run its own migrations to make the database ready for us.  I've added it to the big todo list in https://github.com/neondatabase/neon/issues/6342 as something to do after we embed migrations in the binary, probably in a couple of weeks.